scripts/configure_pyproject_toml.py

```diff
@@ -122,6 +122,18 @@ def parse_args():
         "scipy-openblas32>=0.3.26",
     ]
 
+    if backend == "lightning_tensor":
+        dependencies += ["cutensor-cu12>=2.0.2"]
+
+    if backend in ("lightning_gpu", "lightning_tensor"):
+        dependencies += [
```

Comment:
We may need to include `nvidia-pyindex` to add the NVIDIA private python package index to the user's environment -- only consider this change if you run into any issues with missing NVIDIA  libraries.

Reply:
It seems ok currently, pip doesn't have any issue pulling these packages.